featuremanagement/opentelemetry/_send_telemetry.py

```diff
@@ -0,0 +1,92 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from logging import getLogger as get_logger, INFO
+from .._models import VariantAssignmentReason
+
+
+_event_logger = get_logger(__name__)
+
+try:
+    from opentelemetry.sdk._logs import LoggingHandler
+
+    HAS_OPENTELEMETRY_SDK = True
+except ImportError:
+    HAS_OPENTELEMETRY_SDK = False
+    _event_logger.warning("opentelemetry-sdk is not installed. Telemetry will not be sent to Open Telemetry.")
+
+try:
+    from azure.monitor.events.extension import track_event as azure_monitor_track_event
+
+    HAS_AZURE_MONITOR_EVENTS_EXTENSION = True
+except ImportError:
+    HAS_AZURE_MONITOR_EVENTS_EXTENSION = False
+
+FEATURE_NAME = "FeatureName"
+ENABLED = "Enabled"
+TARGETING_ID = "TargetingId"
+VARIANT = "Variant"
+REASON = "VariantAssignmentReason"
+
+EVENT_NAME = "FeatureEvaluation"
+
+_event_logger.propagate = False
```

Comment:
What's the purpose of this line?

Reply:
It makes sure that the logging is handled by the LoggingHandler on line 47, and not any other. This makes sure open telemetry gets our log. App Insights does the same thing.